CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Implement remotewrite logic
```

Comment:
Could you add more here to explain what remote write is and what this PR really does for reference in the future ?